docs/stark-build/NG_CLI_BUILD_CUSTOMIZATIONS.md

```diff
@@ -117,9 +117,17 @@ Webpack plugins as well as some utils in order to leverage some functionality to
 
 This is the list of utils and plugins used by the Stark-Build package:
 
+### Loaders
+
+### [tslint-loader](https://github.com/wbuchwalter/tslint-loader)
+
+This loader [lints](https://palantir.github.io/tslint/) your TypeScript files based on the configuration in your `tslint.json`.
+
+_Because at this moment disabling `typeCheck` in this loader drasticly improves build time we opted to overwrite and disable all [tslint rules](https://palantir.github.io/tslint/rules/) that require Type Info._
+
 ### Plugins
 
-#### <a name="define-plugin"></a>[DefinePlugin](https://webpack.js.org/plugins/define-plugin)
```

Comment:
This `<a>` element should not be removed because it's a link that is referenced in `ENVIRONMENTS.md` 😉

Reply:
I've double checked and by default the titles in the MD files are considered as links, so in this case the "DefinePlugin" section title creates this link `https://github.com/NationalBankBelgium/stark/blob/master/docs/stark-build/NG_CLI_BUILD_CUSTOMIZATIONS.md#defineplugin`... so I think we can remove the `<a>` element but then we should adapt the link in the `ENVIRONMENTS.md`